src/move-counter/move-counter.html

```diff
@@ -0,0 +1,39 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+
+<dom-module id="move-counter">
+  <template>
+    <style>
+      :host {
+        margin: 0px 1em;
+        color: #fff;
+        font-size: 1.3em;
+      }
+    </style>
+    <div class="counter">Moves: [[moves]]</div>
+  </template>
+
+
+  <script>
+    /**
+     * @customElement
+     * @polymer
+     */
+    class MoveCounter extends Polymer.Element {
+      static get is() {
+        return 'move-counter';
+      }
+      static get properties() {
+        return {
+          moves: {
+            type: Number
+          }
+        };
+      }
+      attached() {
+        this.addEventListener('card-moved', this.moves++);
```

Comment:
![image](https://user-images.githubusercontent.com/581096/28545006-461512a0-7093-11e7-98da-19fea5ca906e.png)

I think you forgot to wrap `this.moves++` in an anonymous function. But it looks like this is actually updated from outside [here](https://github.com/zacharytamas/bakers-dozen/pull/7/files#diff-396d7749118bf5df655db5c1d5dbac0bR72).